utils/netutils/netutils.go

```diff
@@ -1079,3 +1091,16 @@ func DelIPRoute(cidr, gw string) error {
 
 	return netlink.RouteDel(&netlink.Route{Dst: dst, Gw: gwIP})
 }
+
+// ValidateBindAddress format in "address:port"
+func ValidateBindAddress(address string) error {
+	addr := strings.Split(address, ":")
+	if len(addr) != 2 {
+		return fmt.Errorf("bind address is not in 'ip:port' format, got %s", address)
+	}
+	port, err := strconv.Atoi(addr[1])
+	if err != nil {
+		return fmt.Errorf("bind port is a integer, got %v", port)
+	}
```

Comment:
Maybe validate that the port is >= 1 and <= 65535 here?

(I don't think we need to support listening on port 0 lol)

Reply:
done